Sources/FrontEnd/Parse/Parser.swift

```diff
@@ -934,8 +934,7 @@ public enum Parser {
     }
   }
 
-  /// Inserts in `state.ast` a subscript implementation declaration introduced with the given
-  /// introducer and body and returns its ID.
```

Comment:
Yeah, that phrasing is pretty awkward, sorry.  Let me try again:

```suggestion
/// Inserts a subscript having the given `introducer` and `body` into `state.ast`, returning its ID.
```